appservice/config.py

```diff
@@ -4,3 +4,6 @@
 ROUTE_BROWSER = '/wss/webconsole-http/v1'
 # per-session endpoint for managed host
 ROUTE_HOST = '/wss/webconsole-ws/v1'
+
+# 3scale URL for local podman deployment; keep in sync with webconsoledot-local.yaml
+URL_PODMAN = 'https://localhost:8443'
```

Comment:
Hmm, maybe as alternative option we can make it an env variable? Then it's in all in one file?

Reply:
Env variable where? in the Makefile? But the only real variable part is the port number. What do you have in mind?

Reply:
Env variable in our yml file.

Reply:
Env var in the yml is awkward to get to from test_basic.py. Env var in Makefile can substitute the port into the yml. It would then require calling `make check` instead of `pytest` or `python3 -m unittest` directly to actually work, but that'd be acceptable.